pkg/referrerstore/oras/oras.go

```diff
@@ -213,7 +210,7 @@ func (store *orasStore) GetBlobContent(ctx context.Context, subjectReference com
 	// add the repository client to the auth cache if all repository operations successful
 	store.addAuthCache(subjectReference.Original, repository, expiry)
```

Comment:
question about this line,  store.addAuthCache, how do we know if ( subjectReference.Original, repository) is not already in the cache?

Reply:
ah got it, if we ever update this code path, i suggest we should rename it to addAuthCacheIfNeeded or something similar.